### ISSUE TITLE
`add Parquet` installs v0.3.2

### ISSUE THREAD

**Issue by ForceBru** [JuliaIO/Parquet.jl#179]
```
(@v1.10) pkg> add Parquet
   Resolving package versions...
   Installed Snappy ───────── v0.3.0
   Installed MemPool ──────── v0.2.0
   Installed Thrift ───────── v0.6.2
   Installed Parquet ──────── v0.3.2
   Installed BinaryProvider ─ v0.5.10
   Installed ProtoBuf ─────── v0.8.0
    Updating `~/.julia/environments/v1.10/Project.toml`
⌃ [626c502c] + Parquet v0.3.2
    Updating `~/.julia/environments/v1.10/Manifest.toml`
  [b99e7846] + BinaryProvider v0.5.10
⌅ [f9f48841] + MemPool v0.2.0
⌃ [626c502c] + Parquet v0.3.2
⌅ [3349acd9] + ProtoBuf v0.8.0
⌅ [59d4ed8c] + Snappy v0.3.0
⌅ [8d9c9c80] + Thrift v0.6.2
        Info Packages marked with ⌃ and ⌅ have new versions available. Those with ⌃ may be upgradable, but those with ⌅ are restricted by compatibility constraints from upgrading. To see why use `status --outdated -m`
    Building Snappy → `~/.julia/scratchspaces/44cfe95a-1eb2-52ea-b672-e2afdf69b78f/25620a91907972a05863941d6028791c2613888e/build.log`
    Building Thrift → `~/.julia/scratchspaces/44cfe95a-1eb2-52ea-b672-e2afdf69b78f/c3dd01c6067985a77fef761839203838ac12825b/build.log`
Precompiling project...
  6 dependencies successfully precompiled in 22 seconds. 283 already precompiled.

(@v1.10) pkg> 
```

This installs a version [from 2020](https://github.com/JuliaIO/Parquet.jl/releases/tag/v0.3.2). At the same time, Pkg knows that a newer version exists _and_ Parquet.jl "may be upgradable", but it doesn't automatically upgrade.

I then have to manually run `up` in the Pkg REPL to update to v0.8.4.

**Comment by cpfiffer** [comment]
Replicated -- I've had this issue as well. My status after adding Parquet:

```
(analysis) pkg> st
Status `~/bang/analysis/Project.toml`
  [6e4b80f9] BenchmarkTools v1.5.0
  [336ed68f] CSV v0.10.13
  [a93c6f00] DataFrames v1.6.1
⌃ [626c502c] Parquet v0.3.2
  [c3e4b0f8] Pluto v0.19.40
  [df47a6cb] RData v1.0.0
  [2913bbd2] StatsBase v0.34.2
  [ac1d9e8a] ThreadsX v0.1.12
Info Packages marked with ⌃ have new versions available and may be upgradable.
```

**Comment by ForceBru** [comment]
@cpfiffer, FWIW, [Parquet2.jl](https://gitlab.com/ExpandingMan/Parquet2.jl) has been working fine for me. Parquet.jl straight up [can't open _any_ files written by Polars](https://github.com/JuliaIO/Parquet.jl/issues/173), for example, but Parquet2.jl didn't have any issues with this. Seems like Parquet.jl is basically abandoned at this point...

**Comment by asinghvi17** [comment]
This particular issue was caused by old compat bounds - fixed by #183
